.pre-commit-config.yaml

```diff
@@ -19,6 +19,7 @@ repos:
         additional_dependencies:
           - mdformat-gfm
           - mdformat-toc
```

Comment:
Do we need this? Docusaurus provides TOC on the right as an in-page navigation.

Reply:
Good question. We could also take it out later after shipping docusaurus if we want to punt this decision.

Reply:
agree. now that I've had coffee, I realize a lot of my comments here can be used as tickets / tasks to address improving all this further after this is merged in.

Reply:
We could probably improve the formatting of the sidebar TOC though: 

![Screen Shot 2022-03-04 at 10 29 18](https://user-images.githubusercontent.com/1139517/156791979-43bab441-c7f3-4844-b924-c662fbf90d40.png)

Reply:
Yeah, the sidebar is an effect of our weird breadcrumb headers, which we'll be able to totally remove after this is pushed in. Also @rogeruiz saving this work for later is a great idea - I'm going to start making issues for some of the things we've mentioned here that way they're tracked somewhere!